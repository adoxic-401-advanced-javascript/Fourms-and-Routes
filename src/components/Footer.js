import React from 'react';

const Footer = () => {
  return (
    <footer>
      <p> site made by maeve griffin using the X-files api </p>
    </footer>
  );
};

export default Footer;
