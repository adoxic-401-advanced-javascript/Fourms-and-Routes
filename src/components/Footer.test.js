import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer Test', () => {
  it('Renders an Footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
