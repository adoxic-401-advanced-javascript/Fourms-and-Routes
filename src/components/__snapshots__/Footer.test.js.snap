// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test Renders an Footer 1`] = `
<footer>
  <p>
     site made by maeve griffin using the X-files api 
  </p>
</footer>
`;
